dftimewolf/cli/recipes/activity_triage.py

```diff
@@ -0,0 +1,52 @@
+"""Collect artifacts from hosts using GRR.
+
+- Collect a predefined list of artifacts from hosts using GRR
+- Process them locally with grep to extract keywords
+"""
+from __future__ import unicode_literals
+
+_short_description = ('Fetches default artifacts from a list of GRR hosts '
+                      'and processes them with grep.')
+
+contents = {
+    'name':
+        'insider_triage',
```

Comment:
Can we call this "artifact_grep" instead, to be a bit more descriptive? Also, GRR can do a grep remotely, using a condition on a file finder flow. It might need some tweaking on the GRR side to do a grep on artifact files though.

Reply:
I've changed to "activity_triage" (like the file name, etc.), I had missed this one bit still. thanks.
I would keep the "triage" part as this recipe is going to be expanded ideally. The goal is that you, the analyst, want to have a quick glimpse on whether there has been activity regarding specific topics/keywords